x-pack/plugins/security_solution/common/endpoint/schema/actions.test.ts

```diff
@@ -759,4 +759,6 @@ describe('actions schemas', () => {
       }).toThrow('[file]: expected value of type [Stream] but got [Object]');
     });
   });
+
+  describe('ScanActionRequestSchema', () => {});
```

Comment:
still working on these?

Reply:
Oh yeah. I need to add these 😅

Reply:
done c44b3e6884fd143679c62b90e66176ac9504b690